frontend/src/sagas/teachers.js

```diff
@@ -78,7 +79,7 @@ export function* removeTeacher({ id }) {
     }
 }
 
-export function* createTeacher(teacher) {
+export function* createTeacher({ teacher }) {
     const results = { ...teacher };
     if (isObjectEmpty(teacher.department) || !teacher.department?.id) {
```

Comment:
Could you use isEmpty from lodash in this case?

Reply:
done